### PR TITLE
Add template diff when reviewing changeset

### PR DESCRIPTION
**Pull request by kainosnoema**
This allows for more thorough review before applying, including any changes that don't show up in the resource or params changes.

screen shot 2018-08-08 at 12 37 26 pm

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -119,11 +119,9 @@ func (c *Client) Get(stackName string) (*StackInfo, error) {
 
 // GetTemplate retrieves a stack's underlying template
 func (c *Client) GetTemplate(stackName string) (string, error) {
-	input := &cf.GetTemplateInput{
-		StackName:     aws.String(stackName),
-		TemplateStage: aws.String(cf.TemplateStageProcessed),
```
Comment (kainosnoema) [inline client/client.go:124]: We didn't seem to be using this processed version, and the original is the one we want to diff... I think.

File: cmd/stacker/commands/commands.go [hunk 2]
```diff
@@ -478,6 +481,18 @@ func review(stacker *client.Client, changeSet *client.ChangeSetInfo) {
 	}
 
 	reviewStackParams(changeSet.Params, stackInfo.Params)
+
+	stackTemplate, err := stacker.GetTemplate(changeSet.StackName)
+	if err != nil {
+		exitWithError(fmt.Errorf("error fetching template for stack %s", changeSet.StackName))
+	}
+
+	changeSetTemplate, err := stacker.GetChangeSetTemplate(changeSet.StackName, changeSet.Name)
```
Comment (kainosnoema) [inline cmd/stacker/commands/commands.go]: These fetches *could* be done in parallel I think to make output faster.

**Comment by kainosnoema** [comment]
@eyeamera updated to handle templates of both formats as raw strings. Also updated the goformation dependency, as the latest has a breaking change.

**Comment by ascot21** [comment]
@eyeamera it'd be great if we got this merged.

**Comment by ascot21** [comment]
@eyeamera I could really use this. Can we merge?

**Pull request merged**